mlflow/tracking/fluent.py

```diff
@@ -1020,3 +1036,26 @@ def _get_experiment_id():
         or _get_experiment_id_from_env()
         or (is_in_databricks_notebook() and get_notebook_id())
     ) or deprecated_default_exp_id
+
+
+def universal_autolog(
+    log_input_example=False, log_model_signature=True
+):  # pylint: disable=unused-argument
+    # getargvalues isnt actually deprecated
+    # https://docs.python.org/3/library/inspect.html#inspect.getargvalues
+    arg_info = inspect.getargvalues(inspect.currentframe())  # pylint: disable=deprecated-method
+    arg_values = {k: v for k, v in arg_info.locals.items() if k in arg_info.args}
+
+    def setup_autologging(module):
+        integration_module_obj = getattr(mlflow, AUTOLOG_INTEGRATIONS[module.__name__])
+        autolog_fn = getattr(integration_module_obj, "autolog")
+        needed_params = list(inspect.signature(autolog_fn).parameters.keys())
```

Comment:
According to the docs (https://docs.python.org/3/library/inspect.html#inspect.signature), this function may fail with a `ValueError` for certain objects / environments. Can we handle the ValueError case?

Reply:
this seems to occur mostly for functions defined in C and imported into python using Cython.

Thus, it should definitely suffice to just catch the ValueError and call with no arguments in that case.